plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tools/edit/commands/SetEditFeatureCommand.java

```diff
@@ -42,6 +46,10 @@
  */
 public class SetEditFeatureCommand extends AbstractCommand implements UndoableMapCommand {
 
+    // The size of the box that is searched.  This is a 7x7 box of pixels by default.  
+    private int SEARCH_SIZE = Platform.getPreferencesService().getInt(
+            ProjectUIPlugin.ID, PreferenceConstants.FEATURE_SELECTION_RADIUS, 7, null);
```

Comment:
Is **7** the default, if nothing is set in PreferenceStore? I guess it makes sense to define a global Default which was:
* `store.setDefault(PreferenceConstants.P_DELETE_TOOL_RADIUS, 6);` or even
* `store.setDefault(PreferenceConstants.FEATURE_SELECTION_RADIUS,6);`
in PreferencesInitializer but is different to this one.